app/controllers/course/assessment/submissions_controller.rb

```diff
@@ -17,6 +17,10 @@ def create
   end
 
   def edit
+    unless @submission.attempting?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.